libraries/AP_Compass/CompassCalibrator.h

```diff
@@ -13,7 +15,8 @@ enum compass_cal_status_t {
     COMPASS_CAL_RUNNING_STEP_ONE=2,
     COMPASS_CAL_RUNNING_STEP_TWO=3,
     COMPASS_CAL_SUCCESS=4,
-    COMPASS_CAL_FAILED=5
+    COMPASS_CAL_FAILED=5,
+    COMPASS_CAL_BAD_ORIENTATION=6,
```

Comment:
Should this not be `COMPASS_CAL_FAILED_BAD_ORIENTATION` (and the MAVLink message should be `MAG_CAL_FAILED_BAD_ORIENTATION`)

Reply:
I wanted to keep it shorter to fit in MP text box